numerical_methods/rungekutta.cpp

```diff
@@ -0,0 +1,108 @@
+/**
+ *  \addtogroup Runge Kutta 4th Method for solving Differential equation
+ *  @{
+ *  \file
+ *  \brief Runge Kutta Method implementation
+ *
+ *  \author [Rudra Prasad Das](http://github.com/rudra697)
+ *
+ *  \details
+ * It solves the unknown value of y 
+ * for a given value of x
+ * only first oreder differential equations
+ *can be solved
+ *  \example
+ *  it solves dy/dx=(x-y)/2 at a 
+ * given x for given initial 
+ * conditions
+ * There can be many such equations 
+ * **/
+#include<iostream> 
+/**
+ * @brief The change() function is used 
+ * to return the updated iterative value corresponding 
+ * to the given function
+ * @param x is the value corresponding to the x coordinate
+ * @param y is the value corresponding to the y coordinate 
+ * @returns the computed function value at that call
+ * **/
+double change(double x, double y) 
+{ 
+	double val=((x - y)/2.0); 
+	return val;
+} 
+/**
+ *  @brief The rungeKutta finds value of y for 
+ * a given x using step size h 
+ * and initial value initial x at initial y. 
+ * @param init_x is the value of initial x and is updated after each call
+ * @param init_y is the value of initial x and is updated after each call
+ * @param x is the given value of x
+ * @param h is the step value 
+ * @returns the value of y at thr required value of x from the initial conditions
+ **/
+double rungeKutta(double init_x, double init_y, double x, double h) 
+{ 
+	 /**
```

Comment:
Comment blocks shouldn't be used inside functions.
Use normal line-comments, **only** inside functions.

Reply:
Added normal line comments